src/bundles/identity.test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @see: https://github.com/ipfs/ipfs-webui/issues/2033
+ *
+ * Enable WebTransport by
+ * 1. adding "/ip4/0.0.0.0/udp/4002/quic/webtransport" to Addresses.Swarm
+ * 2. enabling transport via ipfs config Swarm.Transports.Network.WebTransport --json true
+ *
+ * Run the tests with
+ *    KUBO_PORT_2033_TEST=5001 npm run test:unit -- --runTestsByPath "test/kubo-webtransport.test.js" --env=./custom-jest-env.js
+ */
+import ipfsHttpModule from 'ipfs-http-client'
+import { createController } from 'ipfsd-ctl'
+
+describe('identity.js', function () {
+  describe('Kubo webtransport fix test', function () {
+    let ipfs
+    let ipfsd
+    beforeAll(async () => {
+      /**
+       * This test allows for a manual run of the Kubo daemon to reproduce and
+       * prove a fix for https://github.com/ipfs/ipfs-webui/issues/2033
+       */
+      const KUBO_PORT = process.env.KUBO_PORT_2033_TEST
+      if (KUBO_PORT == null) {
```

Comment:
I don't think this needs to be in upper case.

```suggestion
      const kuboPort = process.env.KUBO_PORT_2033_TEST
      if (kuboPort == null) {
```

Also, can we do:



```js
      if (KUBO_PORT === null) {
```

or 

```js
      if (!KUBO_PORT) {
```

Reply:
These aren't the same:

- `foo == null` is true is `foo` is `null` or `undefined`
- `foo === null` is true is `foo` is `null`
- `!foo` is true is `foo` is falsy, so `null`, `undefined`, `0`, `''`, or `false`

So `if (KUBO_PORT === null) {` will be `true` if `KUBO_PORT` is `undefined` which is probably not what you meant.

Reply:
@whizzzkid it's pretty idiomatic to assign env vars to UPPER_SNAKE_CASE.

Reply:
@achingbrain what I meant was:

```js
KUBO_PORT === undefined
```

instead of `null`, since `process.env` is an `Object` and this value is never assigned, it will always be `undefined` I cannot think of an example when it would be `null`. However, it's non-blocker just me nit-picking. Also,

> `!foo` is true is `foo` is falsy, so `null`, `undefined`, `0`, `''`, or `false`

is probably what we want unless we allow binding to port 0 🤔.

@SgtPooki env vars in UPPER_SNAKE_CASE makes sense but isn't this a `const` which has been copied over from the env var?